### ISSUE TITLE
Austin-tui won't launch on Windows

### ISSUE THREAD

**Issue by cool-RR** [P403n1x87/austin-tui#3]
Installed using `pip install austin-tui`. 

Versions:
```
austin-python==0.1.0
austin-tui==0.1.3
```

```
c:\Users\Administrator\Desktop>austin-tui.exe -p 8600
🏁 Starting the Austin TUI ...
❌ Austin failed to start. Please make sure that the Austin binary is
available from the PATH environment variable and that the command line
arguments that you have provided are correct.

c:\Users\Administrator\Desktop>python -m austin_tui -p 8600
🏁 Starting the Austin TUI ...
❌ Austin failed to start. Please make sure that the Austin binary is
available from the PATH environment variable and that the command line
arguments that you have provided are correct.
C:\Program Files\Python38\lib\site-packages\austin_tui\__main__.py:137: DeprecationWarning: Task.all_
tasks() is deprecated, use asyncio.all_tasks() instead
  for task in asyncio.Task.all_tasks():
C:\Program Files\Python38\lib\site-packages\austin_tui\__main__.py:140: DeprecationWarning: Task.all_
tasks() is deprecated, use asyncio.all_tasks() instead
  pending = [task for task in asyncio.Task.all_tasks() if not task.done()]
```

**Comment by cool-RR** [comment]
Ah, I realize the problem now. It may have been obvious to you as the developer that Austin and austin_tui are two separate entities, but it won't be obvious to the user, for whom your software is a very small part of their lives. I suggest you include a URL for Austin installation in that error message.

**Comment by P403n1x87** [comment]
Addressed by #7

**Comment by cool-RR** [comment]
Better, but I recommend that the error message will say it's a separate component than Austin TUI and link to the installation. I would have probably missed the readme changed you added.
